integration-libs/s4-service/order/components/order-summary/service-details-card.module.ts

```diff
@@ -10,10 +10,17 @@ import { CommonModule } from '@angular/common';
 import { I18nModule } from '@spartacus/core';
 import { ServiceDetailsCardComponent } from './service-details-card.component';
 import { OrderOutlets } from '@spartacus/order/root';
+import { ServiceOrderOverviewComponentService } from './service-order-overview-component.service';
+import { OrderOverviewComponentService } from '@spartacus/order/components';
 
 @NgModule({
   imports: [CardModule, CommonModule, I18nModule],
   providers: [
+    ServiceOrderOverviewComponentService,
+    {
+      provide: OrderOverviewComponentService,
+      useClass: ServiceOrderOverviewComponentService,
```

Comment:
[MINOR]
In general, let's use `useExisting` instead of `useClass`, for more straightforward extensibility for customers.

Thanks to this, it doesn't matter if they over-provide:
- `provide: OrderOverviewComponentService` OR
- `provide: ServiceOrderOverviewComponentService`
.. their cutomization will work.

Otherwise (if you leave `useClass`), a customer can only customize the service, by:
- `provide: OrderOverviewComponentService`

Reply:
understood, done !!